oak_runtime/src/node/http/tests.rs

```diff
@@ -15,25 +15,37 @@
 //
 
 use super::*;
+use crate::{node::Node, LabelReadStatus::Success};
 use maplit::hashmap;
 use oak_abi::{
-    label::Label,
+    label::{confidentiality_label, tls_endpoint_tag, Label},
     proto::oak::application::{
         node_configuration::ConfigType, ApplicationConfiguration, HttpServerConfiguration,
         NodeConfiguration,
     },
+    Handle,
 };
 use std::fs;
 
 static LOCAL_CA: &str = "../examples/certs/local/ca.pem";
 static GCP_CA: &str = "../examples/certs/gcp/ca.pem";
 
-/// A simple Oak node that responds with 200 (OK) to every request it receives
-struct OakNodeForTest;
+fn get_label(runtime: &RuntimeProxy, handle: Handle) -> Option<Label> {
+    match runtime
+        .get_serialized_channel_label(handle, std::usize::MAX)
```

Comment:
Maybe we should have a method that gets the actual label, without serializing and deserializing, and without having to pass capacity, which is irrelevant, could you create an issue to track?